src/formatter.js

```diff
@@ -0,0 +1,70 @@
+const MESSAGE = Symbol.for('message');
+// eslint-disable-next-line import/no-extraneous-dependencies
+const yamlifyObject = require('yamlify-object');
+// eslint-disable-next-line import/no-extraneous-dependencies
+const yamlifyColors = require('yamlify-object-colors');
+
+const utils = require('./utils');
+const Message = require('./message');
+const Colorizer = require('./colorizer');
+const defaultErrorColors = require('./colors');
+
+const { format } = require('winston');
+
+module.exports = function WinstonConsoleFormatter(options = {}) {
+  options = Object.assign({}, options, {
+    prefix: '',
+    postfix: '',
+    stackTrace: true,
+    colors: defaultErrorColors,
+    types: yamlifyColors,
+  });
+  return format(info => {
```

Comment:
This API is a little bit different to the logform API. I think we should aim to blend in with the logform loggers. For example, this is `format.json`:

```js
module.exports = format((info, opts = {}) => {
  info[MESSAGE] = jsonStringify(info, opts.replacer || replacer, opts.space);
  return info;
});
```

The object that is passed to the formatter when it is created (e.g. `new transports.Console({ formatter: format.json({ space: true }) })`) is passed as the second argument to all calls to your format fn.

Reply:
Yea, I like that a lot

I'll make sure to refer back to https://github.com/winstonjs/logform#understanding-formats when I do it, I don't think i reviewed the docs, just copied an example